controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -408,6 +408,10 @@ def test_get_array_fc_wwns_fail_with_client_exception(self):
         with self.assertRaises(ClientException):
             self.array.get_array_fc_wwns(None)
 
+    def test_get_array_fc_wwns_returns_empty_list(self):
+        self.client_mock.get_host.side_effect = NotFound("404")
+        self.assertListEqual(self.array.get_array_fc_wwns(None), [])
```

Comment:
```suggestion
        self.assertEqual(self.array.get_array_fc_wwns(None), [])
```

https://docs.python.org/3/library/unittest.html#unittest.TestCase.addTypeEqualityFunc
>Note that it’s usually not necessary to invoke these methods directly

Reply:
done

Reply:
thanks. feel free to change existing code (not mandatory in this PR of course)